### ISSUE TITLE
Build error with gcc 10.1.0 and qt 5.15.0

### ISSUE THREAD

**Issue by gizlu** [dogecoin/dogecoin#1642]
### Build error
```
qt/trafficgraphwidget.cpp: In member function ‘void TrafficGraphWidget::paintPath(QPainterPath&, QQueue<float>&)’:                      
qt/trafficgraphwidget.cpp:53:9: error: invalid use of incomplete type ‘class QPainterPath’                                              
   53 |         path.moveTo(x, YMARGIN + h);                                                                                            
      |         ^~~~                                                                                                                    
In file included from /usr/include/qt/QtGui/qbrush.h:49,                                                                                
                 from /usr/include/qt/QtGui/qpalette.h:46,                                                                              
                 from /usr/include/qt/QtWidgets/qwidget.h:48,
                 from /usr/include/qt/QtWidgets/QWidget:1,
                 from qt/trafficgraphwidget.h:8,
                 from qt/trafficgraphwidget.cpp:5:
/usr/include/qt/QtGui/qmatrix.h:54:7: note: forward declaration of ‘class QPainterPath’
   54 | class QPainterPath;
      |       ^~~~~~~~~~~~
qt/trafficgraphwidget.cpp:57:13: error: invalid use of incomplete type ‘class QPainterPath’
   57 |             path.lineTo(x, y);
      |             ^~~~
In file included from /usr/include/qt/QtGui/qbrush.h:49,
                 from /usr/include/qt/QtGui/qpalette.h:46,
                 from /usr/include/qt/QtWidgets/qwidget.h:48,
                 from /usr/include/qt/QtWidgets/QWidget:1,
                 from qt/trafficgraphwidget.h:8,
                 from qt/trafficgraphwidget.cpp:5:
/usr/include/qt/QtGui/qmatrix.h:54:7: note: forward declaration of ‘class QPainterPath’
   54 | class QPainterPath;
      |       ^~~~~~~~~~~~
qt/trafficgraphwidget.cpp:59:9: error: invalid use of incomplete type ‘class QPainterPath’
   59 |         path.lineTo(x, YMARGIN + h);
      |         ^~~~
In file included from /usr/include/qt/QtGui/qbrush.h:49,
                 from /usr/include/qt/QtGui/qpalette.h:46,
                 from /usr/include/qt/QtWidgets/qwidget.h:48,
                 from /usr/include/qt/QtWidgets/QWidget:1,                                                                              
                 from qt/trafficgraphwidget.h:8,                                                                                        
                 from qt/trafficgraphwidget.cpp:5:                                                                                      
/usr/include/qt/QtGui/qmatrix.h:54:7: note: forward declaration of ‘class QPainterPath’                                                 
   54 | class QPainterPath;                                                                                                             
      |       ^~~~~~~~~~~~                                                                                                              
qt/trafficgraphwidget.cpp: In member function ‘virtual void TrafficGraphWidget::paintEvent(QPaintEvent*)’:                              
qt/trafficgraphwidget.cpp:106:22: error: aggregate ‘QPainterPath p’ has incomplete type and cannot be defined                           
  106 |         QPainterPath p;                                                                                                         
      |                      ^                                                                                                          
qt/trafficgraphwidget.cpp:113:22: error: aggregate ‘QPainterPath p’ has incomplete type and cannot be defined                           
  113 |         QPainterPath p;                                                                                                         
      |                      ^                                                                                                          
make[2]: *** [Makefile:7075: qt/libdogecoinqt_a-trafficgraphwidget.o] Error 1
make[2]: Leaving directory '/home/gizlu/tmp/dogecoin-qt/src/dogecoin-1.14.2/src'
make[1]: *** [Makefile:9719: all-recursive] Error 1
make[1]: Leaving directory '/home/gizlu/tmp/dogecoin-qt/src/dogecoin-1.14.2/src'
make: *** [Makefile:685: all-recursive] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
```
### Proposed fix

Include `<QPainterPath>` in src/qt/trafficgraphwidget.h - Works for me

**Comment by patricklodder** [comment]
Thanks! I'm tagging this for 1.14.3 as we need to do something about the QT packaged dependencies in there.

**Comment by patricklodder** [comment]
Fixed with #1655
